lua/wikis/leagueoflegends/Infobox/Team/Custom.lua

```diff
@@ -67,13 +67,9 @@ function CustomTeam:createRegion(region)
 	return remappedRegion and self:createRegion(remappedRegion) or regionData
 end
 
----@return string?
+---@return Widget
 function CustomTeam:createBottomContent()
-	return Template.expandTemplate(
-		mw.getCurrentFrame(),
-		'Upcoming and ongoing tournaments of',
-		{team = self.name}
-	)
+	return UpcomingTournaments{name = self.name}
```

Comment:
if we support onlyHighlightOnValue then we shou allow passing along `onlyHighlightOnValue = self.args.onlyHighlightOnValue` here

for cs we might even want to pass along `highlightOnAnyValue`
for val, ow, apex it isn't relevant due to their highlicghtCond overwrites